keras_ocr/recognition.py

```diff
@@ -8,7 +8,8 @@
 import numpy as np
 import cv2
 
-from . import tools
+from . import tools, config
+config.configure()
```

Comment:
I'm a little hesitant to have a side effect like this occur as part of just importing the module. Can you confirm that it is necessary to do this here and that we can't make it something people call at the top of their own code? That way, if folks are already doing their own config and using these environment variables their own way, this won't result in unexpected behavior. So the README would have a section saying something like:

> To apply these environment variables, do `keras_ocr.config.configure()`

Reply:
I will check it out, thanks for the feedback.